Mathlib/Algebra/Lie/Submodule.lean

```diff
@@ -757,16 +771,40 @@ theorem gc_map_comap : GaloisConnection (map f) (comap f) := fun _ _ ↦ map_le_
 
 variable {f}
 
+theorem map_inf_le :
+    (N ⊓ N₂).map f ≤ N.map f ⊓ N₂.map f := by
```

Comment:
Does the corresponding submodule result prove this in a single line?

Reply:
Yes but it seems `Submodule.map_inf_le` is missing so I've added it and realise that the proof in both cases is of course just `Set.image_inter_subset`.